modules/@apostrophecms/asset/lib/webpack/apos/webpack.scss.js

```diff
@@ -35,6 +36,7 @@ module.exports = (options, apos) => {
                 postcssOptions: {
                   plugins: [
                     [
+                      replaceViewportUnitsPlugin(),
```

Comment:
the plugin should only be enabled if `apos.asset.options.breakpointPreviewMode?.enable === true`

Reply:
good thing I asked your review!